hipcub/include/hipcub/backend/rocprim/device/device_scan.hpp

```diff
@@ -50,15 +55,15 @@ class DeviceScan
                             size_t &temp_storage_bytes,
                             InputIteratorT d_in,
                             OutputIteratorT d_out,
-                            int num_items,
+                            size_t num_items,
                             hipStream_t stream = 0,
-                            bool debug_synchronous = false,
-                            size_t size_limit = size_t(std::numeric_limits<int>::max()) + 1)
+                            bool debug_synchronous = false)
     {
         return InclusiveScan(
             d_temp_storage, temp_storage_bytes,
             d_in, d_out, ::hipcub::Sum(), num_items,
-            stream, debug_synchronous, size_limit
+            stream, debug_synchronous,
+            HIPCUB_DEVICE_SCAN_SIZE_LIMIT
```

Comment:
There's a build failure with the latest change, InclusiveScan doesn't have the size_limit parameter anymore.  we don't need HIPCUB_DEVICE_SCAN_SIZE_LIMIT here anymore right?  Since we are passing it directly to rocPRIM in line 90?